stories/CaseBlockStyled.stories.ts

```diff
@@ -22,7 +22,7 @@ export default {
 }
 
 const CaseBlockTemplate = `
-  <flow-builder-sidebar-editor-container :block="activeBlock">
+  <flow-builder-sidebar-editor-container v-if="activeFlow" :block="activeBlock">
```

Comment:
I think we've already got something like this partially covered from inside of `FlowBuilderSidebarEditorContainer.vue`, like:
```
<flow-builder-container v-if="block || flow">
```

Are we duplicating the logic? Could be nice to not have to remember to add this `v-if` on new block types.

Reply:
For some reason this wasn't quite working for me. Let me try again...

Reply:
Okay, pushed a fix for this:

https://github.com/FLOIP/flow-builder/pull/35/files#diff-130b7357fda7a41c13a1ec6be92137441c79e5a81a4c8ca6445ad1c1cd75ae9fR75

This crashes if no flow is yet present. Slightly unclear why this wasn't an issue before...

Reply:
This link doesn't end up landing me on anything for me 😛 !

Reply:
I think my question is whether we should update the guard that already exists and would only need a change in one place, rather than adding an additional guard to every story?

Reply:
Yup, exactly. Not sure why that link doesn't load. Let me try again:

https://github.com/FLOIP/flow-builder/pull/35/files#diff-130b7357fda7a41c13a1ec6be92137441c79e5a81a4c8ca6445ad1c1cd75ae9fR85

That link works for me to show what I needed to make the existing guard work. This allowed me to delete all changes to stories I had made previously which you can now see in the diff (which no longer includes any stories)